test/concat.js

```diff
@@ -3,8 +3,6 @@ var {expect} = require('chai');
 var assert = require('assert');
 
 describe('concat', function() {
-    this.timeout(250);
```

Comment:
What was the purpose of `this.timeout(250)` before? Was it setting a timeout for this particular test (i.e. fail the test if it doesn't run in under 250 milliseconds)?

Reply:
It's part of my change to set 2000ms as a global timeout. Initially this line increased the timeout from default 200ms, but after my change, its meaning was the opposed. Still double checking the actual value of the 2000ms change.